modules/stanford_profile_styles/lib/scss/paragraph-types/wysiwyg/_buttons.scss

```diff
@@ -17,21 +17,30 @@
     @include modular-spacing('margin-bottom', 0);
   }
 
-  // If they're in the same paragraph, get them to stack.
-  [class*="su-button"] {
-    display: block;
-    width: max-content;
-  }
+  // Allow the power user to stack buttons with .su-stack-button class.
+  .su-stack-button {
```

Comment:
Would it be a good idea to add this to the drop select options in the wysiwyg or does it not function well?

Reply:
@cjwest ^

Reply:
@sherakama, 
I didn't know it was an option! That would be great!